app/src/androidTest/java/com/github/se/cyrcle/ui/profile/CreateProfileScreenTest.kt

```diff
@@ -70,7 +70,7 @@ class CreateProfileScreenTest {
 
     composeTestRule.onNodeWithTag("AuthenticateButton", useUnmergedTree = true).performClick()
 
-    verify(mockNavigationActions).navigateTo(TopLevelDestinations.MAP)
+    verify(mockNavigationActions).navigateTo(Screen.TUTORIAL)
```

Comment:
Making sure I get it: the MockNavigationActions here now goes to the tutorial on first sign in instead of the map, I get that. But I don't get why you straight up removed the test. Is there no way to simulate the already signed-in version of this, which should still lead to the map? Is this test replaced by anything somewhere else?

Reply:
What you call "first sign in" in in reality "create account", which you can only do once per Google account, so the next time, when the user sign in, it goes from *Sign In* to *Map*, whereas this test is checking *CreateProfile* to *Tutorial*

Reply:
Thanks for clarifying. As per my original question, was the original test straight-up removed?

Reply:
No test was removed, only the destination was changed.

Reply:
Understood!